README.md

```diff
@@ -32,6 +32,7 @@
 
 - [ava-fast-check](https://github.com/dubzzz/ava-fast-check) - Property based testing for AVA based on fast-check
 - [fast-check-io-ts](https://github.com/giogonzo/fast-check-io-ts) - io-ts codec to fast-check arbitrary mapping
+- [mock-data-gen](https://github.com/kaeluka/mock-data-gen) - Map io-ts to fast-check arbitraries with flexible support for extending generated data.
```

Comment:
Please follow alphabetical order

Reply:
I've amended the PR (and also moved one other line to be in alphabetical order) :+1: